packages/medusa/src/api-v2/admin/campaigns/[id]/route.ts

```diff
@@ -17,3 +18,26 @@ export const GET = async (req: MedusaRequest, res: MedusaResponse) => {
 
   res.status(200).json({ campaign })
 }
+
+export const POST = async (req: MedusaRequest, res: MedusaResponse) => {
+  const updateCampaigns = updateCampaignsWorkflow(req.scope)
```

Comment:
question: @carlos-r-l-rodrigues, would it be possible to allow for a simplified workflow execution, e.g.:
```ts
import { updateCampaignsWorkflow } from "@medusajs/core-flows"

const { result, error } = await updateCampaignsWorkflow.run({
    input: { campaignsData },
    context: { manager },
    throwOnError: false,
})
```

So, container injection happens automagically, by using whatever default container is available. If none, we throw.

And avoid "calling" the workflow import before running it. I.e. going to `.run()` directly (this is nitpicking).

Reply:
The default container is composed of the modules only. but yes we could probably move the container to the run method